src/include/function/comparison/vector_comparison_functions.h

```diff
@@ -196,40 +196,50 @@ struct ComparisonFunction {
 };
 
 struct EqualsFunction {
+    static constexpr const char* name = "EQUALS";
+
     static inline function_set getFunctionSet() {
-        return ComparisonFunction::getFunctionSet<Equals>(common::EQUALS_FUNC_NAME);
+        return ComparisonFunction::getFunctionSet<Equals>(name);
     }
 };
 
 struct NotEqualsFunction {
+    static constexpr const char* name = "NOT_EQUALS";
+
     static inline function_set getFunctionSet() {
-        return ComparisonFunction::getFunctionSet<NotEquals>(common::NOT_EQUALS_FUNC_NAME);
+        return ComparisonFunction::getFunctionSet<NotEquals>(name);
     }
 };
 
 struct GreaterThanFunction {
+    static constexpr const char* name = "GREATER_THAN";
+
     static inline function_set getFunctionSet() {
-        return ComparisonFunction::getFunctionSet<GreaterThan>(common::GREATER_THAN_FUNC_NAME);
+        return ComparisonFunction::getFunctionSet<GreaterThan>(name);
     }
 };
 
 struct GreaterThanEqualsFunction {
+    static constexpr const char* name = "GREATER_THAN_EQUALS";
+
     static inline function_set getFunctionSet() {
-        return ComparisonFunction::getFunctionSet<GreaterThanEquals>(
-            common::GREATER_THAN_EQUALS_FUNC_NAME);
+        return ComparisonFunction::getFunctionSet<GreaterThanEquals>(name);
     }
 };
 
 struct LessThanFunction {
+    static constexpr const char* name = "LESS_THAN";
+
     static inline function_set getFunctionSet() {
-        return ComparisonFunction::getFunctionSet<LessThan>(common::LESS_THAN_FUNC_NAME);
+        return ComparisonFunction::getFunctionSet<LessThan>(name);
     }
 };
 
 struct LessThanEqualsFunction {
+    static constexpr const char* name = "LESS_THAN_EQUALS";
+
     static inline function_set getFunctionSet() {
```

Comment:
You can remove the inline keyword, since all methods defined in class body are implicity inlined